documentation/docs/examples/cad-colors.mdx

```diff
@@ -15,17 +15,25 @@ Reveal supports updating the color of individual objects by providing [tree indi
 To update the colors of all nodes, loop over all tree indices in the model and update the colors
 accordingly:
 
-<!-- TODO: implement eficcient coloring fn and update these examples -->
-
 ```jsx runnable
 model.iterateNodesByTreeIndex(treeIndex => {
-  model.setNodeColorByTreeIndex(treeIndex, 127, 127, 127);
+  const value = treeIndex % 200;
+  model.setNodeColorByTreeIndex(treeIndex, 200, value, 200);
 });
 ```
 
+> ** New in 1.1.0 **
+
+Notice how colors are applied in batches using this approach. A more performant choice when a single color is 
+to be applied to all nodes is to use `Cognite3DModel.setAllNodeColors`:
+
+```jsx runnable
+model.setAllNodeColors(127, 127, 127);
+```
+
 ## Reset node colors
 
-Color overrides can be overridden using `Cognite3DModel.resetNodeColorByTreeIndex`, e.g.
+Color overrides can be reset using `Cognite3DModel.resetNodeColorByTreeIndex`, e.g.
```

Comment:
But can't we have some reset function as well? A bit weird that we have setAll but not clearAll.

Reply:
We have `resetAllNodeColors`?

Reply:
Sorry, my bad. There is resetAllNodeColors below already.